src/main/java/org/reaktivity/reaktor/internal/router/Target.java

```diff
@@ -170,6 +170,14 @@ private boolean handleWriteSystem(
         switch (msgTypeId)
         {
         case FlushFW.TYPE_ID:
+            handled = streamsBuffer.test(msgTypeId, buffer, index, length);
+            break;
+        case WindowFW.TYPE_ID:
+            if (ReaktorConfiguration.DEBUG_BUDGETS)
+            {
+                System.out.format("[%d] Window ring buffer \n", System.nanoTime());
+            }
+
```

Comment:
This debug output should be either removed or aligned with the output for other `DEBUG_BUDGETS` debug output.